src/components/Button.js

```diff
@@ -2,15 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name } = props;
+  const { name, color, wide } = props;
```

Comment:
You can directly destructure the argument: `const Button = ({ name, color, wide }) => {`.